### ISSUE TITLE
Cite Erlay paper in README

### ISSUE THREAD

**Issue by real-or-random** [sipa/minisketch#25]

**Comment by michaelfolkson** [comment]
Agree, the Erlay paper should be in the references. I was going through them and 8 and 9 return 404s.

https://people.xiph.org/~greg/mempool_sync_relay.txt

https://people.xiph.org/~greg/efficient.block.xfer.txt

Are these hosted anywhere else or should they be deleted from the references?

**Comment by chris-belcher** [comment]
https://web.archive.org/web/20160607023408/https://people.xiph.org/~greg/mempool_sync_relay.txt

https://web.archive.org/web/20160607023333/https://people.xiph.org/~greg/efficient.block.xfer.txt
